app/controllers/conversations_controller.rb

```diff
@@ -19,8 +19,16 @@ def index
     render json: current_user.conversations
   end
 
+  def show
+    render json: conversation
+  end
+
   private
 
+  def conversation
```

Comment:
define this method as a helper method so you can use it in the view https://apidock.com/rails/ActionController/Helpers/ClassMethods/helper_method

Reply:
Same as above, rails api only. No views.